src/librustc/ty/layout.rs

```diff
@@ -610,27 +615,25 @@ impl<'tcx> LayoutCx<'tcx, TyCtxt<'tcx>> {
                 tcx.intern_layout(LayoutDetails {
                     variants: Variants::Single { index: VariantIdx::new(0) },
                     fields: FieldPlacement::Array {
-                        stride: element.size,
+                        stride: element.pref_pos.stride(),
```

Comment:
Should be `element.pref_pos.size`? This was previously not strided.

Reply:
I think I changed this because the value it is assigned to is literally called "stride", but due to the errors, I will change this back.